nise/__main__.py

```diff
@@ -196,12 +196,12 @@ def add_azure_parser_args(parser):
         help="Azure container to place the data.",
     )
     parser.add_argument(
-        "-v2",
-        "--version-two",
-        dest="version_two",
+        "-rg",
+        "--resource-group",
+        dest="resource_group",
```

Comment:
This is changing the interface. Do we have a deprecation period for removing existing command line options? This will break anyone depending on these command line options.

Reply:
This is a good point and I suggest we discuss it.

Reply:
After some discussion with the team, we voted to remove `-v2` flag.